src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/CreateExpiredFunds.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE PROCEDURE [employer_financial].[CreateExpiredFunds]
+	@accountId BIGINT,
+	@expiredFunds [employer_financial].[ExpiredFundsTable] READONLY
+AS
+	INSERT [employer_financial].[TransactionLine] (AccountId, DateCreated, TransactionDate, TransactionType, Amount)
```

Comment:
It might be worth adding a unique index to the `TransactionLine` table on the `TransactionDate` and `TransactionType` columns where `TransactionType = 5` to ensure we never get any dupes.

Reply:
Added, with columns ordered to help speed up the `GetExpiredFunds` stored proc.